### PR TITLE
feat: Use usize instead of u8

### PR DESCRIPTION
**Pull request by kdheepak**
Here's the minimum number of changes for converting from u8 to usize, if you are interested in it.

### CODE REVIEW

File: src/graph/calc.rs [hunk 1]
```diff
@@ -210,8 +210,11 @@ fn update_commit_line<'a>(
...
     commit_line_state: &mut [Option<&'a CommitHash>],
     target_commit_hashes: &[&CommitHash],
-) -> u8 {
-    let mut min_pos_x = commit_line_state.len();
+) -> usize {
+    if commit_line_state.is_empty() {
+        return 0;
+    }
+    let mut min_pos_x = commit_line_state.len().saturating_sub(1);
     for target_hash in target_commit_hashes {
```
Comment (lusingander) [inline src/graph/calc.rs:218]: It doesn't seem necessary for this PR.

**Review by lusingander — COMMENTED** [review]
This seems fine, so if you don't mind, could you please fix it so that it passes CI?

**Comment by lusingander** [comment]
https://github.com/lusingander/serie/issues/4

File: src/graph/calc.rs (hunk 1 above)
Comment (kdheepak) [reply src/graph/calc.rs]: I've resolved this change back to the `master`.

**Comment by kdheepak** [comment]
I rebased on top of the latest `master` and resolved the merge conflicts. This should be good to merge now.

**Review by lusingander — APPROVED** [review]
Thank you!!

**Pull request merged**